cmd/kava/opendb/opendb_rocksdb.go

```diff
@@ -42,11 +43,15 @@ const (
 
 	defaultColumnFamilyName = "default"
 
-	maxOpenFilesDBOptName          = "max_open_files"
-	maxFileOpeningThreadsDBOptName = "max_file_opening_threads"
+	enableMetricsOptName             = "rocksdb.enable_metrics"
```

Comment:
nit: do we need to prefix these with `rocksdb`? Would prefer following existing code convention for the sake of code consistency / coherent read-ablity

Reply:
Update: after reviewing more of the files I see this is the current convention, ignore me

Reply:
these are app.toml properties accessed through AppOptions, yes? i think they need to be namespaced to something. makes sense to me to put the rocksdb-specific metrics settings with the rocksdb configuration 
could also see them going under the existing `telemetry`, but separating into a `rockdb` object we control seems good to

do you have an example of the convention you're talking about, @galxy25? it's posisble i'm not understanding what these constants are

Reply:
looks like this in `app.toml`:
```toml
[rocksdb]
enable_metrics = true
report_metrics_interval_secs = 150

max_open_files = 142
write_buffer_size = 67108142
```

Reply:
@pirtleshell here https://github.com/Kava-Labs/kava/pull/1692/files#diff-ad965588a6a423832eb132b027de107d69a3a60df814a509acb511a50a6726f1R32 Where `rocksdb` prefix is also used, versus the variables that previously existed in this `opendb_rocksdb.go` file e.g. `write_buffer_size`

Reply:
I think I have to use `dashes` instead of `underscores`:
- `max_open_files = 142` -> `max-open-files = 142`

Initially I used `underscores` because `rocksdb` uses them, but anyway we probably have to follow our convention

Reply:
@galxy25 fyi: https://github.com/Kava-Labs/kava/pull/1692/files#diff-ad965588a6a423832eb132b027de107d69a3a60df814a509acb511a50a6726f1R32 - this is not `appOpts`, this is `rocksdb properties`, just want to make sure we're on the same page. Their names are hardcoded on rocksdb-level

Reply:
fixed

Reply:
as of now looks like this:
```toml
[rocksdb]
enable-metrics = true
report-metrics-interval-secs = 15

max-open-files = 142
write-buffer-size = 67108142
```